callbacks/create.go

```diff
@@ -126,6 +127,22 @@ func Create(config *Config) func(db *gorm.DB) {
 		insertOk := err == nil && insertID > 0
 
 		if !insertOk {
+			if db.IgnoreLastInsertIDWhenNotSupport {
```

Comment:
I think this feat is to allow users to skip updating the model's primary key. We don't need to pay attention to the internal logic of the driver. Whether to skip `LastInsertID` can be config by the user or the driver itself.